state/futures/actions.ts

```diff
@@ -281,15 +282,18 @@ export const fetchIsolatedMarginTradePreview = createAsyncThunk<
 	async (sizeDelta, { dispatch, getState, extra: { sdk } }) => {
 		const marketInfo = selectMarketInfo(getState());
 		const account = selectFuturesAccount(getState());
+		const marketPrice = selectMarketPrice(getState());
+		const skewAdjustedPrice = selectSkewAdjustedPrice(getState());
+
 		if (!account) throw new Error('No account to fetch orders');
 		if (!marketInfo) throw new Error('No market info');
 		const leverageSide = selectLeverageSide(getState());
 		try {
 			const preview = await sdk.futures.getIsolatedTradePreview(
 				marketInfo?.market,
 				sizeDelta,
-				marketInfo?.priceOracle,
-				marketInfo?.price,
+				skewAdjustedPrice,
+				marketPrice,
```

Comment:
Looks like these are backwards

```suggestion
				marketPrice,
				skewAdjustedPrice,
```